IBM/CAST
# csmdb_update: fn_csm_switch_inventory_history_dump now handles NULL f…

…ields without creating duplicate records in the history table.


# Pull Request Title
bills_fn_csm_switch_inventory_history_dump_update

## Purpose
Updated the function to handle NULL fields without creating duplicate records in the history table.

- [ ] @pdlun92 to review (check regression)
- [ ] @NickyDaB to review with basic API testing + code review.

## How to Test
INSERT a record or records with potential 'NULL' field values.
THEN make sure there are non duplicated records in the history table when updating (Running inventory collection - If no fields have changed).
